tests/test_jacobi.c

```diff
@@ -0,0 +1,99 @@
+#include "../src/jacobi.h"
```

Comment:
Relative imports will definitely break in the future. Please use #include <jacobi.h> and make sure that when compiling you use -I../src/ (Which should already be present in the makefile)